providers/aws/resources/aws_s3.go

```diff
@@ -205,28 +202,33 @@ func (a *mqlAwsS3Bucket) policy() (*mqlAwsS3BucketPolicy, error) {
 	policy, err := svc.GetBucketPolicy(ctx, &s3.GetBucketPolicyInput{
 		Bucket: &bucketname,
 	})
-	if err != nil {
-		if isNotFoundForS3(err) {
-			return emptyAwsS3BucketPolicy(a.MqlRuntime)
-		}
+	if err != nil && !isNotFoundForS3(err) {
```

Comment:
perhaps it would be useful to set the plugin state here? `plugin.StateIsNull | plugin.StateIsSet`

Reply:
Nice, I learned something new today. Thanks @vjeffrey 👍🏽